metabase/metabase
# pivot exports by default

### Description

[Slack convo](https://metaboat.slack.com/archives/C064QMXEV9N/p1729176141796499) 
Makes "Keep data pivoted" enabled by default on pivot table exports.
<img width="294" alt="Screenshot 2024-10-18 at 3 41 37 PM" src="https://github.com/user-attachments/assets/e63e4100-4731-45af-8cf5-ec1105c91faf">

### How to verify

- Create a pivot table and save question
- Try to download results -> Ensure "Keep data pivoted" is enabled

### Checklist
